### PR TITLE
feat: add refuse logic to revoke a tx as co-signer

### PR DESCRIPTION
**Pull request by marc0olo**

### CODE REVIEW

**Comment by marc0olo** [comment]
FYI -> I missed the README. will provide a new commit with updated readme

**Comment by marc0olo** [comment]
@thepiwo I additionally updated the version number to 2.0.0

File: README.md [hunk 1]
```diff
@@ -43,11 +43,16 @@ By performing a [GaAttachTx](https://aeternity.com/protocol/generalized_accounts
...
-    - the signature is checked against the stored hash
+- `refuse(hash)`
+    - params
+        - `hash` the tx-hash of the meta transaction that should be authorized
+    - can be called to refuse the currently proposed tx
 - `revoke(hash)`
     - params
         - `hash` the tx-hash of the meta transaction that should be authorized
     - can be called to explicitely revoke the currently proposed tx
+        - no check if the caller is the proposer of the tx
```
Comment (hanssv) [inline README.md]:
```suggestion
        - now checks if the caller is the proposer of the tx
```

Comment (marc0olo) [reply README.md]: yeah that was a bit of an unfortunate statement 😅

should have written sth. like "passes without an additional check if the caller is proposer of the tx" or so 🤔

**Review by thepiwo — APPROVED** [review]
looks good to me

File: README.md (hunk 1 above)
Comment (marc0olo) [reply README.md]: I consider this resolved now :)

**Review by thepiwo — APPROVED** [review]

**Pull request merged**